addons/sale_pdf_quote_builder/wizards/sale_pdf_quote_builder_dynamic_fields_wizard_views.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="pdf_quote_builder_dynamic_fields_wizard" model="ir.ui.view">
+        <field name="name">sale.pdf.quote.builder.dynamic.fields.wizard.view.form</field>
+        <field name="model">sale.pdf.quote.builder.dynamic.fields.wizard</field>
+        <field name="arch" type="xml">
+            <form string="Configure Dynamic Fields">
```

Comment:
is this string displayed somewhere ?
Unless I'm wrong, only the action names will be displayed